### PR TITLE
isort setup.py

### PR DESCRIPTION
**Pull request by brettlangdon**
We are using `isort` on other files, not sure why `flake8-isort` hasn't complained about `setup.py`, but figured we can do it manually.

(ok, the semi-real reason is my editor auto-isorts so I have to ignore these changes whenever I modify `setup.py`)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,13 +2,14 @@
 import platform
 import sys
 
-from setuptools import setup, find_packages, Extension
-from setuptools.command.test import test as TestCommand
-
 # ORDER MATTERS
 # Import this after setuptools or it will fail
```
Comment (Kyle-Verhoog) [inline setup.py:6]: This is probably why we don't isort 😆

**Review by Kyle-Verhoog — DISMISSED** [review]

File: setup.py (hunk 1 above)
Comment (brettlangdon) [reply setup.py:6]: oh, I cannot read.... I wonder if I can `# noqa` them 🤔

**Comment by brettlangdon** [comment]
I wonder if the better approach is to just disable `isort` for this entire file? the skips seem alright, but wonder if they will be error prone with ordering?

**Comment by Kyle-Verhoog** [comment]
@brettlangdon we skip it already: https://github.com/DataDog/dd-trace-py/blob/8891132131f45a14371871aad9f9e736e278c19e/pyproject.toml#L14

**Comment by brettlangdon** [comment]
hmm... so weird... what is wrong with my editor 🤔

**Pull request closed without merging**